<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatGUI</class>
 <widget class="QDialog" name="ChatGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatGUI</string>
  </property>
  <widget class="QTextBrowser" name="log_textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>461</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>491</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="msg_textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>461</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="send_pushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLabel" name="peerName_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
